app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,214 @@
+<resources>
+    <string name="app_name">Corona Alert</string>
+
+    <string name="all_next">Next</string>
+    <string name="all_cancel">Cancel</string>
+    <string name="all_thl_image_description">Finnish Institute for Health and Welfare</string>
+    <string name="all_today">Today</string>
+    <string name="all_yesterday">Yesterday</string>
+    <string name="all_app_additional_info">More info on Corona Alert</string>
+    <string name="all_system_enable">Enable</string>
+
+    <string name="diagnosis_title">Report infection</string>
+    <string name="diagnosis_body1">If you have tested positive for SARS-CoV-2, you can anonymously report your diagnosis using the code supplied by your healthcare provider or THL (Finnish Institute for Health and Welfare).</string>
```

Comment:
```suggestion
    <string name="diagnosis_body1">If you have tested positive for COVID-19, you can anonymously report your diagnosis using the code supplied by your healthcare provider or THL (Finnish Institute for Health and Welfare).</string>
```

Reply:
Excellent suggestions! Thank you for corrections. In this case though I think the original makes more sense. According to official guidelines https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/naming-the-coronavirus-disease-(covid-2019)-and-the-virus-that-causes-it SARS-CoV-2 is the virus, while COVID-19 is the disease. The tests check for the presence of the virus, the disease may or may not be developed like in the case of asymptomatic carriers.

Reply:
In common language I suppose it is referred to as the "covid test", so lets go with COVID-19